src/helper/remote_config/protocol/tuf/serializer.cpp

```diff
@@ -0,0 +1,164 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+
+#include <rapidjson/document.h>
+#include <rapidjson/prettywriter.h>
+
+#include "../../../json_helper.hpp"
+#include "../cached_target_files.hpp"
+#include "serializer.hpp"
+
+namespace dds::remote_config::protocol::tuf {
+
+const char *product_to_string(product product)
```

Comment:
Instead of returning `const char *`, this can return a `rapidjson::GenericStringRef<char> `, although there's nothing really wrong with this.